fcrepo-kernel-modeshape/src/test/java/org/fcrepo/kernel/modeshape/FedoraResourceImplTest.java

```diff
@@ -255,6 +255,33 @@ public void testGetLastModifiedDate() {
     }
 
     @Test
+    public void testTouch() {
+        // test existing JCR_LASTMODIFIED
+        final Calendar someDate = Calendar.getInstance();
+        someDate.add(Calendar.DATE, -1);
+        try {
```

Comment:
Instead of these `try/catch` blocks (here and at line:272), is there any reason not to just add a `throws` clause on the `testTouch()` method declaration? It is not clear that we will every run into an exception case when setting up the mocks.